.github/workflows/pythonapp.yml

```diff
@@ -8,6 +8,7 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
+      fail-fast: false
```

Comment:
Why do you want to keep on running the tests if one has failed?

Reply:
Ah, because of two reasons:
* Multiple tests might fail.
* A test might fail only on a particular os/python version combination.

Both points are obscured when fail-fasting.